records_mover/creds/base_creds.py

```diff
@@ -1,7 +1,9 @@
 from db_facts.db_facts_types import DBFacts
 from .database import db_facts_from_env
 from typing import TYPE_CHECKING, Iterable, Union, Optional
-from records_mover.mover_types import NotYetFetched
+from records_mover.mover_types import PleaseInfer
```

Comment:
I standardized on the latter here; the difference in semantics was so subtle as to be meaningless for the uses here.